deps/picotls/t/cli.c

```diff
@@ -492,7 +492,7 @@ int main(int argc, char **argv)
             setup_log_event(&ctx, optarg);
             break;
         case 'v':
-            setup_verify_certificate(&ctx);
+            setup_verify_certificate(&ctx, optarg);
```

Comment:
What is the argument that should be specified by existing users of `-v` option?

I think we can break backward compatibility of cli (its primary use-case is for testing as indicated by the directory in which the source file exists), but we need to support existing use-case, hence the question.

Reply:
the argument is the location to client provided root CA so that certificate can be verified. I think that it can be optional so that default trust store can be used.

Reply:
Added another -V with provided root CA file and kept -v using default.